html/index.html

```diff
@@ -12,38 +12,35 @@
 
 
 var width = 960,
-    height = 1160;
+    height = 600;
 
 var svg = d3.select("body").append("svg")
     .attr("width", width)
     .attr("height", height);
 
-var path = d3.geo.path()
-    // .projection(projection);
+var projection = d3.geo.albers()
```

Comment:
Honestly I started with the projection Mike Bostock uses for the UK in http://bost.ocks.org/mike/map/, and adjusted the center and scale to make London fill the map. Probably a better set of parameters would be `.center([0, 51.49]).rotate([0.1, 0])`